app/lib/dttp/mappable.rb

```diff
@@ -59,7 +59,7 @@ def dttp_qualification_aim_id(training_route)
     end
 
     def dttp_school_id(urn)
-      Dttp::School.find_by(urn: urn)&.dttp_id
+      Dttp::School.active.find_by!(urn: urn)&.dttp_id
```

Comment:
Should we raise an explicit error here that we could catch in the worker to avoid the retries spamming sentry?

Reply:
Yup i'll put it in a follow up PR 👍